src/app/analytics/analytics.factory.js

```diff
@@ -482,13 +482,13 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
         // Parse the cart object of the last purchase
         const transactionCart = JSON.parse(localStorage.getItem('transactionCart'))
         // The purchaseId number from the last purchase
-        const lastTransactionId = localStorage.getItem('transactionId')
+        const lastTransactionId = sessionStorage.getItem('transactionId')
```

Comment:
`sessionStorage` is not currently available to this factory I think.

Reply:
It seems to work. Is it because ```$window``` is exposed in the factory? The other places we use sessionStorage just do ```this.sessionStorage = $window.sessionStorage```
<img width="457" alt="Screen Shot 2020-11-23 at 10 01 01 AM" src="https://user-images.githubusercontent.com/39680460/99977649-ecf60580-2d72-11eb-8ee6-cd19669d1c37.png">

Reply:
Interesting. Not sure why.

Reply:
`localStorage` and `sessionStorage` are both JS globals because everything on the `window` object is considered to be global. Best practices in AngularJS seem to suggest you get these from `$window`. I think that's to make mocking in tests a little easier. But idk if it matters too much.